backend/src/schema/types/type/Badge.gql

```diff
@@ -3,8 +3,6 @@ type Badge {
   type: BadgeType!
   status: BadgeStatus!
   icon: String!
-  #createdAt: DateTime
-  #updatedAt: DateTime
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-11-18T15:15:03Z" title="Monday, November 18th 2019, 4:15:03 pm +01:00">Nov 18, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -3,8 +3,6 @@ type Badge {
   type: BadgeType!
   status: BadgeStatus!
   icon: String!
-  #createdAt: DateTime
-  #updatedAt: DateTime
```


I understand why you want to clean this up, but you haven't lost hope on refactoring to use `DateTime`, right?
have you seen https://github.com/neo4j-graphql/neo4j-graphql-js/issues/272 ??